workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestManagerStatus/util/patchRequestStatus.ts

```diff
@@ -14,7 +15,8 @@ import {Status} from '../../../common/utils/constants/status';
 const patchRequestStatus = async (
 	mdfRequestStatus: LiferayPicklist,
 	mdfRequestId: string,
-	mdfReqToActs?: MDFRequestActivityDTO[]
+	mdfReqToActs?: MDFRequestActivityDTO[],
+	mdfReqToMDFClms?: MDFClaimDTO[]
```

Comment:
Maybe you can do the request inside this patchRequestStatus function.

```suggestion
```

Reply:
Tried to do this, but got this error after change.
![image (1)](https://github.com/liferay-customer-delivery/liferay-portal/assets/90978981/331147aa-66cf-4b6e-9179-053ae1e2bbef)

Reply:
Done!